### PR TITLE
[Vectorized](udaf) fix java-udaf couldn't get jar core dump

### PR DESCRIPTION
**Pull request by zhangstar333**
# Proposed changes

**If be couldn't get jar, when  call agg create function will be failed, so exec_place is nullptr cause core dump** 

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.55 seconds
 load time: 430 seconds
 storage size: 17173121695 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221118120916_clickbench_pr_48700.html

**Comment by HappenLee** [comment]
Should do the check and return error Status in create function be called !

File: be/src/vec/aggregate_functions/aggregate_function.h [hunk 1]
```diff
@@ -198,6 +199,9 @@ class IAggregateFunction {
 
     virtual DataTypePtr get_serialized_type() const { return std::make_shared<DataTypeString>(); }
 
+    //only used to check java udaf jar is valided, so it's could retrn error for user before open phase
+    virtual Status check_udaf(const TFunction& fn) { return Status::OK(); }
```
Comment (HappenLee) [inline be/src/vec/aggregate_functions/aggregate_function.h:203]: should not be virtual function

**Review by HappenLee — DISMISSED** [review]
LGTM

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**